### PR TITLE
[CH-191] Support generate exec 

### PR DESCRIPTION
**Pull request by taiyang-li**
Changelog category (leave one):
- New Feature

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Support generate exec. close https://github.com/Kyligence/ClickHouse/issues/191

> Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by kyligence-git** [comment]
Can one of the admins verify this patch?

File: utils/local-engine/Parser/SerializedPlanParser.h [hunk 1]
```diff
@@ -18,6 +18,7 @@
 #include <substrait/plan.pb.h>
 #include <Common/BlockIterator.h>
 #include <Core/SortDescription.h>
+#include <mutex>
```
[the remark this answers is not in the document]
Comment (taiyang-li) [reply utils/local-engine/Parser/SerializedPlanParser.h]: It's for debug. removed now.

**Comment by taiyang-li** [comment]
This pr must be merged with https://github.com/oap-project/gluten/pull/574 together.

**Comment by taiyang-li** [comment]
@zzcclp @liuneng1994 Can you review this pr please? Thanks

**Comment by taiyang-li** [comment]
test this please with 574.

**Comment by taiyang-li** [comment]
test this please with 574

**Comment by taiyang-li** [comment]
test this please with 574

**Comment by taiyang-li** [comment]
test this please with 574

**Comment by taiyang-li** [comment]
test this please with 574

**Comment by taiyang-li** [comment]
test this please with 574

**Comment by taiyang-li** [comment]
test this please with 574

**Comment by taiyang-li** [comment]
test this please with 574

**Pull request merged**